diff --git a/debian/rules b/debian/rules
index e84669f..b32d22b 100755
--- a/debian/rules
+++ b/debian/rules
@@ -15,7 +15,7 @@ debian/cainteoir-data.sharedmimeinfo: data/mime/packages/cainteoir-data.xml
 override_dh_auto_configure:
 	dh_testdir
 	./autogen.sh
-	dh_auto_configure -- --disable-update-mime-database $(shell dpkg-buildflags --export=configure)
+	dh_auto_configure -- --disable-update-mime-database
 
 configure: override_dh_auto_configure
 
